modules/webapp/src/main/elm/Messages/Basics.elm

```diff
@@ -87,7 +87,7 @@ de =
     , outgoing = "Ausgehend"
     , tags = "Tags"
     , items = "Dokumente"
-    , submit = "Absenden"
+    , submit = "Speichern"
```

Comment:
Just want to mention that this is used in various places. The idea with "Submit/Absenden" is to be more generic. But I can't recall a specific situation right now, where "Speichern" wouldn't fit well. So I think we go with this and if a case comes up where it doesn't fit, we can separate translations.